packages/main/src/events/CallbackEvent.ts

```diff
@@ -5,6 +5,7 @@ import { TsLogger } from '../utils/logger'
 
 export enum PresentationStatus {
   OK = 'ok',
+  WAITING = 'waiting',
```

Comment:
```suggestion
  CONNECTED = 'connected',
```

I think probably "Connected" is a bit more meaningful here and gives us the opportunity of adding other states we may need in the future (e.g. "CODE_SCANNED" or something)